src/HomePage/Skills/Skills.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./Skills.scss";
+
+const Skills = () => {
+  return (
+    <div className="skills">
+      <div>
+        <i className="icons devicon-github-plain" />
+      </div>
+      <div>
+        <i className="icons devicon-html5-plain" />
+      </div>
+      <div>
+        <i className="icons devicon-javascript-plain" />
+      </div>
+      <div>
+        <i className="icons devicon-react-original" />
+      </div>
+      <div>
+        <i className="icons devicon-webpack-plain" />
+      </div>
+      <div>
+        <i className="icons devicon-css3-plain" />
+      </div>
+      <div>
+        <i className="icons devicon-gitlab-plain" />
+      </div>
+      <div>
+        <i className="icons devicon-sass-original" />
+      </div>
+    </div>
```

Comment:
This is a lot of repeated code. Could you create an array of icon names and loop over it and render the icons like that? Eg
```
const icons = ["devicon-github-plain", "devicon-html5-plain", ...]

const Skills = () => {
  return (
    {icons.map(icon => (
      <div>
        <i className={`icons ${icon}`} />
      </div>
    ))}
  )
};
```